cmd/boulder-janitor/job.go

```diff
@@ -166,7 +166,8 @@ func (j batchedDBJob) RunForever() {
 		for {
 			lastID, err := j.getWork(work, id)
 			if err != nil {
-				j.log.Err(err.Error())
+				j.log.Errf("error getting work for %q from ID %d: %v",
```

Comment:
Ever so slight nit: the formatter for `err.Error()` can just be `%s` since the return type is string. If you want to print the error type I think you'd just want the var to be `err`.

Reply:
Good point, using just the string is probably fine here. I changed the format specifier to `%s` in 2116d4a.